dstep/translator/Type.d

```diff
@@ -350,9 +352,10 @@ body
         {
             if (children[dimension].kind == CXCursorKind.CXCursor_IntegerLiteral)
             {
+                auto token = tokenInsideSquareBrackets(cursor);
                 auto expansions = context.macroIndex.queryExpansion(children[dimension]);
 
-                if (expansions.length == 1)
+                if (expansions.length == 1 && token.kind == TokenKind.identifier)
```

Comment:
The spelling of `token`, which you added, seems to be correct. At least for this case, not sure if the rest of the tests would pass.